docker/pytorch-aarch64/scripts/build-pytorch.sh

```diff
@@ -33,6 +35,19 @@ git checkout v$version -b v$version
 git submodule sync
 git submodule update --init --recursive
 
+# Clone torch xla
```

Comment:
This XLA build will execute for every build now, regardless of whether the XLA backend is desired. Could you guard the XLA build here with a new build flag? It should be possible to supply a `--xla` option to `build.sh` and pass that all the way down to here (as per the `--onednn acl_threadpool` option in the TF build).